templates/tfengine/components/cicd/terraform.tfvars

```diff
@@ -98,5 +96,6 @@ envs = [
         }
     }
   },
-  {{- end}}
+  {{- end}} 
 ]
+{{hclField . "grant_automation_billing_user_role" -}}
```

Comment:
Move this up. Putting this at the end especially below the big github and csr block will make it easily ignored.

Reply:
Done